cpp/tests/reductions/scan_tests.cu

```diff
@@ -344,7 +349,8 @@ TEST_F(ScanStringTest, SkipNA)
   cudf::test::strings_column_wrapper expected2(exact.begin(), exact.end(), out_b.begin());
   std::unique_ptr<cudf::column> col_out;
   //skipna=false
-  CUDF_EXPECT_NO_THROW(col_out = cudf::experimental::scan(col_nulls, scan_op::MAX, true, false));
+  CUDF_EXPECT_NO_THROW(col_out = cudf::experimental::scan(col_nulls, 
+    cudf::experimental::make_max_aggregation(), true, false));
   if(false) {
```

Comment:
I assume this is _debug_ code that might get activated on occasion for debugging purposes. If so, perhaps pre-processor directive (`#ifdef _DEBUG`, etc.) would convey better its intent. 

Or, if it's a runtime decision (for example, by passing certain runtime arguments to the test) it should depend on a flag. As it is, it's just inactive code.

Reply:
If it's no longer needed, delete it.

Reply:
removed.